src/cider/nrepl/middleware/track_state.clj

```diff
@@ -31,8 +31,8 @@
   [m]
   (and (:macro m)
        (:arglists m)
-       (not (:style/indent m))
-       (not (:indent m))
+       (not (contains? m :style/indent))
+       (not (contains? m :indent))
```

Comment:
Is this a backward-compability wart? I've never seen anybody use `:indent`.

Reply:
Me neither, but from memory, it's always been there internally